src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -110,6 +112,7 @@ def __init__(
             resampled_data_recv: A receiver that delivers samples with a
                 given sampling period.
             sampling_period: The sampling period.
+            resampling_period: The resampling period.
```

Comment:
Yeah, it confused me too. In the resampler I call the input sampling period "source sampling period", but I guess it's different, because the resampler doesn't control this, while this class controls both, right?

Reply:
correct, this class controls both.
Also keep in mind the `resampling_period` or `output_sampling_period` is optional.
Anyway it's fine with me renaming both input parameters and updating the documentation too.

Reply:
I've initially done the renaming but then I thought it will be more suitable to pass the `ResamplerConfig` as it is more flexible to set different functionality and it also provides default parameters which fit well the current use case of the MovingWindow

Reply:
The way I see it is the `ResamplerConfig` provides all the default parameters suitable for the `MovingWindow` given the current use-case, and I found it useful and helpful the fact that the user can provide another resampling function through the `ResamplerConfig` along with the resampling period if needed.
Anyway I'm happy to address it is the way you think it is the best as you have more experience with the users of the SDK. @leandro-lucarella-frequenz @matthias-wende-frequenz any thought on this matter?

Reply:
I was thinking this was something else.  Based on the clarifications from the meeting,  I'm ok with going forward with this.